# Breast_Cancer_Tcells_SCRNA_Analysis_Project

## Overview

This project focuses on the analysis of publicly available single-cell RNA sequencing (scRNA-seq) data related to breast cancer T cells. The primary objectives of this project were to perform data preprocessing, normalization, cluster analysis, and identify genes associated with T cells within the breast cancer dataset.

## Data Source

The scRNA-seq data used in this project was obtained from publicly available sources. Please refer to the data source or repository for detailed information on the dataset.

## Project Workflow

1. **Data Preprocessing**: Prior to analysis, the raw scRNA-seq data underwent a series of preprocessing steps. This included quality control, filtering out low-quality cells, and removing unwanted sources of variation.

2. **Normalization**: To ensure that the data was comparable across all cells, normalization was performed. This step is crucial for accurate downstream analysis.

3. **Clustering**: Clustering analysis was carried out to group cells with similar gene expression profiles. This allowed for the identification of distinct cell populations, including T cells, within the breast cancer dataset.

4. **Gene Identification**: Genes associated with T cells were identified through differential gene expression analysis. This step aimed to highlight genes that are specifically expressed in T cells or show significant differences in expression compared to other cell types.

![](https://github.com/JoshTjan/Breast_Cancer_Tcells_SCRNA_Analysis_Project/blob/main/BC_Cluster_Labeled.png) ![](https://github.com/JoshTjan/Breast_Cancer_Tcells_SCRNA_Analysis_Project/blob/main/CD8%2Bmarker_dot.png)
